src/Services/SubscriptionService.php

```diff
@@ -121,6 +121,14 @@ public function delete($subscriptionId)
         return $this->client()->delete('subscriptions/{subscriptionId}', ['subscriptionId' => $subscriptionId]);
     }
 
+    /**
+     * @param string $subscriptionId
```

Comment:
What does this operation return?

Reply:
Nothing, my mistake. Thanks for the tip

Reply:
Alright, you've fixed another mistake, my question is about the return of `void` request though. According the API definitions returns "no content" (doubtful solution), but this method returns something.

Reply:
Added return type